linera-core/src/client/mod.rs

```diff
@@ -1132,79 +1181,133 @@ where
         // Retrieve newly received certificates from this validator.
         let query = ChainInfoQuery::new(chain_id).with_received_log_excluding_first_n(tracker);
         let info = remote_node.handle_chain_info_query(query).await?;
+        let mut tracker = MultichainTracker::new(tracker);
+        let remote_log = info.requested_received_log.clone();
+        let remote_node_chains_view = info.requested_received_log.into_iter().fold(
+            BTreeMap::<ChainId, Vec<BlockHeight>>::new(),
+            |mut chain_to_info, entry| {
+                chain_to_info
+                    .entry(entry.chain_id)
+                    .or_default()
+                    .push(entry.height);
+                chain_to_info
+            },
+        );
+
+        // Collect all certificates that have been successfully processed.
```

Comment:
```suggestion
        /// Collects all certificates that have been successfully processed.
```

Reply:
This is a comment _within_ a function – should it have triple-`/` comment as well?

Reply:
Sorry! :facepalm: 
No, of course not.